core/src/providers/anthropic.rs

```diff
@@ -271,6 +273,41 @@ impl TryFrom<&ChatMessage> for AnthropicChatMessage {
 
 // Tools.
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+#[serde(rename_all = "lowercase")]
+pub enum AnthropicToolChoiceType {
+    Auto,
+    Any,
+    Tool,
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+pub struct AnthropicToolChoice {
+    r#type: AnthropicToolChoiceType,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    name: Option<String>,
+}
+
+impl FromStr for AnthropicToolChoice {
+    type Err = ParseError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
```

Comment:
we don't want to parse the any ?

Reply:
we don't support it officially so for now no.

Reply:
> we don't support it officially so for now no.

Agree that we don't support, but anthropic.rs doesn't really know that right ?

Reply:
fair